src/reducers/systemIntakeReducer.ts

```diff
@@ -156,13 +157,31 @@ function systemIntakeReducer(
       return {
         ...state,
         notes: [
-          ...state.notes,
           {
             ...action.payload,
             createdAt: DateTime.fromISO(action.payload.createdAt)
-          }
+          },
+          ...state.notes
         ]
       };
+    case fetchIntakeNotes.TRIGGER:
+      return {
+        ...state,
+        notes: []
```

Comment:
This was the idea I had of "clearing" out comments. When we fetch notes by intake ID, we clear the notes before the request is initiated.